### ISSUE TITLE
Anti-ribosomal P protein antibody positivity

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#11622]
**Preferred term label:**
Anti-ribosomal P protein antibody positivity

**Synonyms**
Anti-ribo P
anti-ribosomal P protein antibody positivity
anti-P Abs
anti-RibP

**Definition (free text, please give PubMed ID)**
Anti-ribosomal P protein antibodies  recognize three specific ribosomal proteins located in the large ribosome's subunit. Anti-ribosomal P protein autoantibody recognize three proteins, P0, P1 and P2.
PMID: 34796669
PMID: 15804709
PMID: 37206021
PMID: 37572300

**Parent term (use hpo.jax.org/app)**
HP:0033476

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
systemic lupus erythematosus

**Your nano-attribution (ORCID)**
0000-0002-3387-1836; 0009-0004-5292-8481

**Comment by pnrobinson** [comment]
@MaudDLVT 
Adding new term: 
- ID: HP:6001538
- label: Anti-ribosomal P protein antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:34796669|PMID:15804709|PMID:37206021|PMID:37572300
- def: The presence of an antibody in the blood circulation that is directed against ribosomal P protein.
- comment: Anti-ribosomal P protein antibodies (anti-ribo P) have been reported as one of the specific autoantibodies in patients with systemic lupus erythematosus (SLE) and has been demonstrated to bind and activate macrophages in vitro. Anti-ribosomal P protein antibodies (Abs) recognize three specific ribosomal proteins located in the large ribosome's subunit. The term anti-ribosomal P protein Abs, often shortened in anti-P Abs, is due to the fact that these three proteins, P0, P1 and P2, of 38, 19 and 17 Kd molecular weight, respectively, are phosphorylated. One of the major points of interest of these autoAbs derives from their high specificity for systemic lupus erythematosus (SLE), their description in other connective tissue diseases (CTDs) being only occasional.
- orcid: orcid:0000-0002-3387-1836
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11622
- synonyms:
